### PR TITLE
Quiz page

### PR DESCRIPTION
**Pull request by talrejah02**
Added quiz page
please review and give suggestions using this live [link](https://deploy-preview-3--playful-bombolone-e2e666.netlify.app/)

### CODE REVIEW

File: src/components/QuizCard/QuizCard.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import "./QuizCard.css"
+
+function QuizCard() {
+  return (
+    <div className='quiz-card flex-col justify-center align-center'>
+        <div className=""><span className='quiz-header'>How to delete a directory in Linux?</span></div>
```
Comment (shuntzhere) [inline src/components/QuizCard/QuizCard.jsx:7]: Classname can be removed, if not used.

File: src/components/QuizCategory/CategoryCards.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    border-radius: 1px;
+    margin-bottom: 0.4rem;
+    color: var(--accent);
+    font-size: 1.2rem;
+    font-weight: 400;
+    transition: 0.4s;
+}
+.cart-btn:hover{
+    background-color:var(--accent) ;
+    color: #191A1A;
```
Comment (shuntzhere) [inline src/components/QuizCategory/CategoryCards.css:33]: Use variables

File: src/components/QuizCategory/CategoryCards.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./CategoryCards.css";
+import { NavLink } from "react-router-dom";
+
+function CategoryCards() {
+  return (
+    <section class="card shadow">
+      <img class="card-img" src="Images/cloud.jpg" alt="" />
```
Comment (shuntzhere) [inline src/components/QuizCategory/CategoryCards.jsx:8]: Give an alt text

**Review by shuntzhere — APPROVED** [review]
Seems good, great work!

**Review by ScoobyDubeyDo — APPROVED** [review]

**Pull request merged**